manifests/calico-etcd.manifest

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
I think the suffix on this file should be `.yaml` not `.manifest` - did you see an example that did otherwise?